sources/de/nw/statewide.json

```diff
@@ -0,0 +1,24 @@
+{
+	"coverage": {
+        "country": "de",
+        "state": "nw"
+    },
```

Comment:
Can we add an ISO code here? Should look like this [based on the coverage docs](https://github.com/openaddresses/openaddresses/blob/master/CONTRIBUTING.md#coverage-object):

```JSON
"ISO 3166": { "alpha2": "DE-NW" }
```